### PR TITLE
re-format the slash returning msg

### PR DESCRIPTION
**Pull request by JisanAR03**
before :
 
image

after : 

image

### CODE REVIEW

**Comment by JisanAR03** [comment]
@arkid15r sir, can you please review this PR

**Comment by Sarthak5598** [comment]
Looks good ! You're really good at formatting these and improving the ux

File: src/sammich/plugins/contributors.py [hunk 1]
```diff
@@ -46,36 +46,63 @@ def format_data(prs, issues, comments):
...
-    max_comments_length = max(len(str(counts["comments"])) for counts in user_data.values())
-
-    table = [
-        f"| {'User':<{max_name_length}} | PRs Merged | Issues Resolved | Comments |",
-        f"|{'-' * (max_name_length + 2)}|:{'-' * (max_prs_length + 2)}:|"
-        f":{'-' * (max_issues_length + 2)}:|:{'-' * (max_comments_length + 2)}:|",
-    ]
+    # Calculate column widths with minimum lengths
+    max_user_length = max(len(user) for user in user_data.keys())
+    max_prs_length = max(max(len(str(counts["prs"])) for counts in user_data.values()), 10)
```
Comment (arkid15r) [inline src/sammich/plugins/contributors.py:51]: Could you explain the `max(max(len(str(` logic here?
Thanks!

File: src/sammich/plugins/contributors.py [hunk 2]
```diff
@@ -46,36 +46,63 @@ def format_data(prs, issues, comments):
...
+    )
+    rows = [header, separator]
+
+    # Formatting each user's data
     for user, counts in user_data.items():
-        table.append(
-            f"| {user:<{max_name_length}} | {counts['prs']:>{max_prs_length}} | "
-            f"{counts['issues']:>{max_issues_length}} | "
-            f"{counts['comments']:>{max_comments_length}} |"
+        row = (
```
Comment (arkid15r) [inline src/sammich/plugins/contributors.py:74]: Avoid introducing a new variable if you use it only once

File: src/sammich/plugins/contributors.py [hunk 3]
```diff
@@ -46,36 +46,63 @@ def format_data(prs, issues, comments):
...
+        channel_id = command._cmd_payload["channel_id"]
+        user = command._cmd_payload["user_id"]
+        prs, issues, comments = fetch_github_data("OWASP-BLT", "BLT")
+        formatted_data = format_data(prs, issues, comments)
+        if not formatted_data:
+            formatted_data = [
+                {"type": "section", "text": {"type": "mrkdwn", "text": "No data available"}}
+            ]
+
+        await self.web_client.chat_postEphemeral(
```
Comment (arkid15r) [inline src/sammich/plugins/contributors.py:106]: You can use higher level [API](https://dondebonair.github.io/slack-machine/api/#machine.plugins.base.MachineBasePlugin.say) for communication (e.g. `self.say()`)

Comment (JisanAR03) [reply src/sammich/plugins/contributors.py:106]: .say() . not allowed to use in rich formatting like block.

File: src/sammich/plugins/contributors.py (hunk 1 above)
Comment (JisanAR03) [reply src/sammich/plugins/contributors.py:51]: basically sir ,this one is for count how much space we should use in each column

**Comment by JisanAR03** [comment]
@arkid15r sir , updated the commit . can you please review for merging

File: src/sammich/plugins/contributors.py (hunk 3 above)
Comment (arkid15r) [reply src/sammich/plugins/contributors.py:105]: The docs say the opposite:

> Send a message to a channel using the WebAPI. Allows for rich formatting using blocks and/or attachments/

Could you provide more details supporting your statement?

**Review by arkid15r — COMMENTED** [review]
LGTM

**Comment by JisanAR03** [comment]
> LGTM

thanks sir 😊

**Pull request merged**